addons/website_slides/static/src/js/slides_course_quiz_question_form.js

```diff
@@ -44,13 +46,43 @@ var QuestionFormWidget = publicWidget.Widget.extend({
         var self = this;
         return this._super.apply(this, arguments).then(function () {
             self.$('.o_wslides_quiz_question input').focus();
+            self.$('.o_wslides_js_quiz_answer_comment > input[type=text]').each(function () {
+                self._bindCommentInputToEvent($(this));
+            });
```

Comment:
It's cleaner if you can bind those in the "evens" parameter (just like the click).

`'change .o_wslides_js_quiz_answer_comment > input[type=text]': '_onCommentChanged'`